### PR TITLE
Rav/feature branch example

### PR DESCRIPTION
**Pull request by raisahv**
blah blahblah

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Person A should modify content here.
 
 ## Person B:
 
-Person B should modify content here.
+hello - raisah was here
```
Comment (krismosk) [inline README.md:11]: I might suggest writing a test for this line of code, :)

**Review by krismosk — CHANGES_REQUESTED** [review]
Submitted

File: README.md (hunk 1 above)
Comment (raisahv) [reply README.md:11]: Thanks.

**Review by raisahv — COMMENTED** [review]
Added a comment

**Pull request merged**